### ISSUE TITLE
I don't understand

### ISSUE THREAD

**Issue by Bl0xxy** [Ceebox/cbLang#16]
Correct me if I'm wrong, but I don't believe this is a coding language but instead something that translates from your syntax to Python.  Is that correct?

**Comment by Maciko84** [comment]
yes. this translates the syntax to python. he explained it in his video: (https://youtu.be/y4gLcYfVCe8)
